tests/cases/string/regexp_match_substring.test

```diff
@@ -2,34 +2,34 @@
 ### SUBSTRAIT_INCLUDE: '/extensions/functions_string.yaml'
 
 # basic: Basic examples without any special cases
-regexp_match_substring('foobarboopzoo'::str, 'o{1,}'::str, 1::i64, 1::i64, 0::i64) = 'oo'::str
```

Comment:
looks like you're removing some function tests and adding others. Why not keep existing and add new ones?

Reply:
understood! will update.